docs/command-line-interface/aserto-cli/control-plane.mdx

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_label: Control Plane
+title: Control Plane
+description: Using the control plane
+---
+
+# Control Plane
+
+Aserto [edge authorizers](/docs/edge-authorizers/overview) can connect to the Aserto control plane to receive policy and
+directory updates and commands. Edge authorizers must use client certificates from satellite connections to connect to
+the control plane. 
+
+## Configuration
+
+The Aserto CLI can be used to configure certificates. The `list-connections` sub-command lists a tenant's existing
+satellite connections:
+
+```
+aserto control-plane list-connections
+```
+
+Each of the listed connections has an `id` field, which can be used to retrieve certificate data, including the
+certificate and private key:
+
+```
+aserto control-plane client-cert <satellite-connection-id>
+```
+
+For more details on how to configure the certificate see the [edge
+authorizers](/docs/edge-authorizers/security-and-management) section of this documentation.
+
+## Commands
+
+To list the edge authorizer instances connected to the control plane:
+```
+aserto control-plane list-instance-registrations
+```
+
```

Comment:
It might be a good idea to show example results for each of the sub-commands.

Reply:
We don't do that anywhere else in the Aserto CLI section, so it seemed out of place to do it only for the control plane.